pypsi/shell.py

```diff
@@ -40,7 +40,7 @@ class Shell(object):
     # pylint: disable=too-many-public-methods
 
     def __init__(self, shell_name='pypsi', width=79, exit_rc=-1024, ctx=None,
-                 features=None):
+                 features=None, completer_delims=" "):
```

Comment:
Since this is a new property, I'd rather not reset every shell's completer delims when they update pypsi. Can you make the default value is `None` and then only call `readline.set_completer_delims()` if `self.completer_delims is not None`

Reply:
Yes, you are right! I have just committed this requested change.